### PR TITLE
Add formula 5.5 from NEN-EN-1993.5:2008

### LINKED ISSUE #194: [✨ Feature request]: Add formula 5.5 from NEN-EN 1993-5
### Description

Add formula 5.5 from NEN-EN 1993-5

### Acceptance Criteria

- Formula 5.5 
- Unit checks
- Docstrings 

### Dependencies

_No response_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by Klaassicking**
## Description

Add formula 5.5 from NEN-EN-1993.5:2008

$V_{pl,Rd} = \frac{A_v f_y}{\sqrt{3} \gamma_{M0}}$

Fixes #194 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: blueprints/codes/latex_formula.py [hunk 1]
```diff
@@ -3,6 +3,30 @@
 from dataclasses import dataclass
 
 
+def fraction(numerator: str | float, denominator: str | float) -> str:
```
Comment (bro-wi) [inline blueprints/codes/latex_formula.py:6]: I would rename this function to be more explicit, e.g. `latex_fraction`

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_5.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    label = "5.5"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        a_v: MM2,
+        f_y: MPA,
+        gamma_m_0: DIMENSIONLESS,
+    ) -> None:
+        """(Vpl,Rd) Calculate plastic shear resistance in [kN/m] based on formula 5.5 from NEN-EN 1993-5:2007(E) art. 5.2.2(4).
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_5.py:25]:
```suggestion
        """(Vpl,Rd) Calculate plastic shear resistance in [kN] based on formula 5.5 from NEN-EN 1993-5:2007(E) art. 5.2.2(4).
```

mm2*N/mm2 / (unitlesss) ==> N

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_5.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        self.a_v: float = a_v
+        self.f_y: float = f_y
+        self.gamma_m_0: float = gamma_m_0
+
+    @staticmethod
+    def _evaluate(
+        a_v: MM2,
+        f_y: MPA,
+        gamma_m_0: DIMENSIONLESS,
+    ) -> KNM:
```
Comment (bro-wi) [inline blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_5.py:46]: This is not a moment

```suggestion
    ) -> KN_M:
```

or even better (IMO)

```suggestion
    ) -> KN:
```

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            (200, 100, -0.5),  # gamma_m_0 is negative
+            (200, 100, 0),  # gamma_m_0 is zero
+        ],
+    )
+    def test_raise_error_when_negative_or_zero_n_t_rd_is_given(self, a_v: float, f_y: float, gamma_m_0: float) -> None:
+        """Test a zero value for parameters a_v, f_y and gamma_m_0."""
+        with pytest.raises(LessOrEqualToZeroError):
+            Form5Dot5PlasticShearResistance(a_v=a_v, f_y=f_y, gamma_m_0=gamma_m_0)
+
+    def test_latex_output(self) -> None:
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py:42]: use parametrize here, now the test possible will fail in the first check and will leave other un noticed. see also Sina's comment on #191 in that regard

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # Example values
+        a_v = 200  # MM2
+        f_y = 100  # MPA
+        gamma_m_0 = 0.5  # DIMENSIONLESS
+        form = Form5Dot5PlasticShearResistance(a_v=a_v, f_y=f_y, gamma_m_0=gamma_m_0)
+
+        # Expected result, manually calculated
+        expected = 40000 / np.sqrt(3) / 1000
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py:22]:
```suggestion
        expected = 40 / np.sqrt(3)
```

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # Example values
+        a_v = 200  # MM2
+        f_y = 100  # MPA
+        gamma_m_0 = 0.5  # DIMENSIONLESS
+        form = Form5Dot5PlasticShearResistance(a_v=a_v, f_y=f_y, gamma_m_0=gamma_m_0)
+
+        # Expected result, manually calculated
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_5.py:21]: not really 'manually' calculated isn't it?

File: tests/codes/test_latex_formula.py [hunk 7]
```diff
@@ -32,3 +32,15 @@ def test_str(self, fixture_latex_formula: LatexFormula) -> None:
         # Expected result
         expected_result = "E = mc^2 = 5*10^2 = 500"
         assert str(fixture_latex_formula) == expected_result
+
+
+def test_latex_fraction() -> None:
+    """Test the latex_fraction function."""
```
Comment (bro-wi) [inline tests/codes/test_latex_formula.py:38]:
```suggestion
    """Test the latex_fraction function."""
```
This name (latex_fraction) is much better, but it it is different from the function you are testing.

**Review by bro-wi — CHANGES_REQUESTED** [review]
Nice work and nice to see you are doing so many PR's!

Main concern from my part is getting all the units correct, because now it is not really consistent IMO.

cheers.

**Review by egarciamendez — APPROVED** [review]
Suggestions from Wichard have been implemented. 😉 🚀

**Pull request merged**